README.rst

```diff
@@ -89,9 +90,8 @@ services`_ offerings.
 * LinkedIn Group - `<https://www.linkedin.com/groups/4877160>`_
 * Google+ - `<https://plus.google.com/b/112856352920437801867/+SaltStackInc/posts>`_
 
-.. _SaltConf: http://www.youtube.com/user/saltstack
```

Comment:
Why remove the YouTube channel link?

Reply:
In my mind it is linked right below in the **Follow SaltStack on** section. Also the link title didn't really represent where you would end up. It might make sense to steer people to watch past SaltConf talks on YouTube but we might want to think on how it should we written; perhaps into the section's dialog.

Reply:
That makes sense. Approved